lib/routes/nautiljon/manga-releases.ts

```diff
@@ -0,0 +1,70 @@
+import { load } from 'cheerio';
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import puppeteer from '@/utils/puppeteer';
+import logger from '@/utils/logger';
+
+const host = 'https://www.nautiljon.com';
+export const route: Route = {
+    path: '/releases/manga',
+    categories: ['reading'],
+    example: '/releases/manga',
```

Comment:
```suggestion
    example: '/nautiljon/releases/manga',
```
example should have namespace prepended